PyPoE/cli/exporter/wiki/parsers/item.py

```diff
@@ -2779,10 +2812,14 @@ def _harvest_plant_booster_extra(self, infobox, base_item_type,
         row_index=True,
     )
 
+    '''
+    This defines the expected data elements for an item class.
+    '''
```

Comment:
It seems to me like we're inside of a method here, which would mean that we should use line comments here instead of docstrings. Can we please change this to be `# This defines the expected data elements for an item class.` insead?

Reply:
It's within a class, not a method.
I'm tweaking it so intellisense will pick up on the docstring.